lib/pdfkit/pdfkit.rb

```diff
@@ -80,6 +88,11 @@ def to_file(path)
     File.new(path)
   end
 
+  def is_windows?
+    require 'rbconfig'
+    !(RbConfig::CONFIG['host_os'] =~ /mswin|mingw|cygwin/).nil?
```

Comment:
Not an expert here, but can I ask why this combination of platform names?

Stackoverflow offers both of these options:

```
/mswin|msys|mingw|cygwin|bccwin|wince|emc/
/cygwin|mswin|mingw|bccwin|wince|emx/ 
```

Can I safely assume that these lists include some really old identifies (MSDOS, eg) and that the three you have are the modern systems?